### DISCUSSION TITLE
Is evobot abandoned or simply stopped updates cause of Summer?

### DISCUSSION THREAD

**Discussion by nikosszzz** [eritislami/evobot#1221] [Q&A] [answered]
Is evobot gonna get any further updates and fixes? Has support stopped or just a summer break?

**Comment by eritislami** [comment, marked as answer]
Hi @nikosszzz it's not abandoned but I can understand why that would seem so, I have been insanely busy and haven't found time to put it the bot, once the summer season has calmed down a bit and when I get some free time I have many things to go through, I just can't invest my time currently due to other high priority matters

**Reply by nikosszzz** [reply]
Ah, alright I see. thank you for the response and the amazing bot music system!

**Reply by rockrevenchy** [reply]
dw, the bot is doing a fantastic job rn as is, take care of urself first!
